static/src/javascripts/bootstraps/enhanced/notifications.js

```diff
@@ -43,37 +48,65 @@ define([
             return modules.getReg().then(function (reg) {return reg.pushManager.getSubscription();});
         },
 
-        configureSubscribeTemplate: function () {
-            var subscribed = modules.checkSubscriptions(),
-                hasNoSubscriptions = modules.subscriptionsEmpty(),
+        init: function() {modules.configureSubscribeButton();
+            if(!modules.hasSubscribed() && !modules.hasDismissedExplainer()) {
+                modules.showExplainer();
+            }
+        },
+
+        configureSubscribeButton: function () {
+            var $follow = bonzo(qwery('.js-live-notifications')[0]),
+                subscribed = modules.checkSubscriptions(),
                 handler = subscribed ? modules.unSubscribeHandler : modules.subscribeHandler,
-                src = template(subscribeTemplate, {
-                    className: hasNoSubscriptions ? '' : 'notifications-subscribe-link--has-subscriptions',
-                    text: subscribed ? 'Unfollow' : 'Follow story',
-                    imgMobile: svgs('notificationsExplainerMobile', ['mobile-only', 'notification-explainer']),
-                    imgDesktop: svgs('notificationsExplainerDesktop', ['hide-on-mobile', 'notification-explainer'])
+                src = template(followLink, {
+                    subscribed: subscribed,
+                    icon: svgs(subscribed ? 'notificationsOff' : 'notificationsOn')
                 });
 
+            $follow.html(src);
+            bean.one($follow[0], 'click', '.js-notifications__button', handler );
```

Comment:
Why are we using `one` here? Can you not unsubscribe and then subscribe again? If so, you would need to checkSubscriptions in your handler, not just on page load.